WordPress/gutenberg
# Post content block: create content select pattern placeholder

## What?
1. Create a placeholder for empty post content blocks on pages and posts.
2. Update the post content placeholder copy and styles for templates.

Using most of @SaxonF's ideas from https://github.com/WordPress/gutenberg/pull/56898 🙇🏻 

## Why?
1. To provide convenient information and prompts to the user facing an empty post content block in the editor.
2. To offer a more appealing placeholder state.

## How?


## Testing Instructions
1. Create a new page in the site editor. Ensure the page's template has a post-content block.
4. Check that the post content block placeholder can open the patterns inserter.
5. Also test the "Start blank" button. The new paragraph will be selected.
6. Edit the page's template.
7. Behold the post content placeholder!
8. Repeat steps 6-7 in the post editor.

## Screenshots or screencast 



https://github.com/WordPress/gutenberg/assets/6458278/d8efbf93-5beb-4f0a-9d3f-2e525bf110c9
